pkg/graph/resolvers/model_plan.go

```diff
@@ -20,7 +20,7 @@ import (
 // TODO Revist this function, as we probably want to add all of these DB entries inthe scope of a single SQL transaction
```

Comment:
```suggestion
// TODO: Revisit this function, as we probably want to add all of these DB entries in the scope of a single SQL transaction
```

Reply:
Hi @OddTomBrooks , that was a note from a previous time, not from this ticket, I don't this is the ticket to address it

Reply:
Fair enough. Bad formatting on TODO: may skip IDEs from picking them up so it threw a large red flag for me.